cfgov/cfgov/urls.py

```diff
@@ -303,6 +303,10 @@
     # Form csrf token provider for JS form submission
     url(r'^token-provider/', token_provider),
 
+    # data-research-api
+    url(r'^data-research/mortgages/api/v1/',
+        include_if_app_enabled('data_research', 'data_research.urls')),
```

Comment:
A great question. We have had a number of APIs that live in external repos, so I was following the pattern of the API being integrated into the app's URL path. But it might make sense to corral all cfgov APIs under one API path separate from the Wagtail tree.

I think a decision about what that namespace would be is in order before decoupling this API from the data_research app. This would affect CCDB5 as well, I would assume.